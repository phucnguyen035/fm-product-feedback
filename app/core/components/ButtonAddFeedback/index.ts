export { default as ButtonAddFeedback } from './ButtonAddFeedback';
