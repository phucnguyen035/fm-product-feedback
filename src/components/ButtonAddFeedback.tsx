import { PlusIcon } from '@heroicons/react/solid';
import Link from 'next/link';

import Button from './Button';

export default function ButtonAddFeedback() {
  return (
    <Link passHref href="/feedback/add">
      <a data-testid="add-feedback">
        <Button className="no-underline" variant="pink" prependIcon={<PlusIcon />}>
          Add feedback
        </Button>
      </a>
    </Link>
  );
}
