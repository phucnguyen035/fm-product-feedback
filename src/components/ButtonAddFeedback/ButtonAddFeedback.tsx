import { PlusIcon } from '@heroicons/react/solid';
import Link from 'next/link';

import { Button } from '../Button';

export default function ButtonAddFeedback() {
  return (
    <Link passHref href="/feedback/add">
      <a>
        <Button className="no-underline" variant="pink" prependIcon={<PlusIcon />}>
          Add feedback
        </Button>
      </a>
    </Link>
  );
}
