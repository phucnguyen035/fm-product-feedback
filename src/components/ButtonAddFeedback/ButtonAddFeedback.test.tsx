import { render } from '@testing-library/react';

import ButtonAddFeedback from './ButtonAddFeedback';

describe('Button Add Feedback', () => {
  it('should render correctly', () => {
    const { container } = render(<ButtonAddFeedback />);

    expect(container).toMatchSnapshot();
  });
});
