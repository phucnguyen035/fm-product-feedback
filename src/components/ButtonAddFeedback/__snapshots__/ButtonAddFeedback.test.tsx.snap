// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Add Feedback should render correctly 1`] = `
<div>
  <a
    href="/feedback/add"
  >
    <button
      class="w-36 rounded-lg py-2.5 px-4 text-body3 font-bold outline-none transition-colors md:w-40 md:text-sm bg-pink hover:bg-pink/80 focus:outline-pink/80 text-gray-light no-underline"
    >
      <div
        class="flex items-center justify-center"
      >
        <span
          aria-label="icon left"
          class="mr-1 h-4 w-4 md:mr-3.5 md:h-5 md:w-5"
        >
          <svg
            aria-hidden="true"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M10 3a1 1 0 011 1v5h5a1 1 0 110 2h-5v5a1 1 0 11-2 0v-5H4a1 1 0 110-2h5V4a1 1 0 011-1z"
              fill-rule="evenodd"
            />
          </svg>
        </span>
        Add feedback
      </div>
    </button>
  </a>
</div>
`;
